lib/cli/src/warn.test.ts

```diff
@@ -0,0 +1,39 @@
+import globby from 'globby';
```

Comment:
any reason to add a dep when we already depend on `glob`?

Reply:
In CLI, there was no dependency. Globby seemed more reliable from my (quick) testing of the two, and supports arrays. This implementation was _lifted_ from the same check in CRA.